src/tutorials/0006-anatomy-of-a-cid/06.md

```diff
@@ -22,7 +22,7 @@ Looking at the results from the CID Inspector tool we can see several parts that
 - `Multicodec`: `code` is the identifier of the codec, in this case `0x70` for `dag-pb`, an IPLD format
 - `Multihash`: breakdown of the multihash into the hashing algorithm used (`18` is the code for `sha2-256`) and the length of the hash (256 bits, which equates to 32 bytes)
 
-From the "Human Readable CID" breakdown, we can see that the original hash of the content, before the appropriate CIDv1 prefixes are added, is `c3c4733ec8affd06cf9e9ff50ffc6bcd2ec85a6170004bb709669c31de94391a`.
+From the "Human Readable CID" breakdown, we can see that the original hash of the content, before the appropriate CIDv1 prefixes are added, is `C3C4733EC8AFFD06CF9E9FF50FFC6BCD2EC85A6170004BB709669C31DE94391A`.
```

Comment:
I think we need to mention the phrase "digest (in hex)" at some point in the text now since it's been added in the screenshot. 

Why is this in caps? You mentioned that it emphasizes that it's a hex value but I don't understand that reference. Has anything changed that means we're incorrectly using lower case earlier in the tutorial?

Reply:
updated